### PR TITLE
WIP Test Jenkins build with MPL3

### PR DESCRIPTION
**Pull request by rozyczko**

### CODE REVIEW

**Comment by rozyczko** [comment]
Ready for testing win

**Comment by rozyczko** [comment]
ready for testing on Win

**Comment by rozyczko** [comment]
ready for testing on Win

**Comment by rozyczko** [comment]
ready for testing on Win

**Comment by rozyczko** [comment]
ready for testing on Win

File: build_tools/conda_qt5_win_commercial.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ dependencies:
  - pyzmq=19.0.1
  - mkl=2020.0
  - pip:
-   - matplotlib==2.2.3
+   - matplotlib==3
```
Comment (pkienzle) [inline build_tools/conda_qt5_win_commercial.yml:28]: Not a specific version? The current release is 3.4.3 from 2021-08-13 on PyPI

**Comment by rozyczko** [comment]
ready for testing on Win

File: build_tools/conda_qt5_win_commercial.yml (hunk 1 above)
Comment (rozyczko) [reply build_tools/conda_qt5_win_commercial.yml]: precisely

**Comment by rozyczko** [comment]
ready for testing on Win

**Comment by rozyczko** [comment]
ready for testing on Win

**Comment by rozyczko** [comment]
ready for testing on Win

**Comment by rozyczko** [comment]
ready for testing on Win

**Comment by rozyczko** [comment]
ready for testing on Win

**Comment by rozyczko** [comment]
ready for testing on Win

**Pull request closed without merging**